### ISSUE TITLE
Can't install via archive file distributed on pypi

### ISSUE THREAD

**Issue by calebrob6** [omry/omegaconf#772]
**Describe the bug**

The `setup.py` script uses `build_helpers/` however `build_helpers/` isn't included in the pypi code archive, so running `python setup.py sdist` from that archive isn't possible.

**Expected behavior**
Expect to be able to install via the code archive uploaded to pypi.

**Comment by Jasha10** [comment]
Thanks for the report. I can reproduce the issue:
```text
$ # Download the archive linked from https://pypi.org/project/omegaconf/#files
$ wget https://files.pythonhosted.org/packages/db/fa/dfcbaffd016aeedd20f6ce800de171e13128490f767e8f68ec3baab21eaa/omegaconf-2.1.0.tar.gz
$ tar -xzf omegaconf-2.1.0.tar.gz
$ cd omegaconf-2.1.0/
$ pip install .
Processing /Users/jasha10/tmp/omegaconf-2.1.0
...
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/jasha10/miniconda3/envs/tmp/bin/python /Users/jasha10/miniconda3/envs/tmp/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py get_requires_for_build_wheel /var/folders/6r/93zx8_1s4jxg484fcjjzgjp40000gn/T/tmpok_6xgs6
       cwd: /private/var/folders/6r/93zx8_1s4jxg484fcjjzgjp40000gn/T/pip-req-build-6jx88eo9
  Complete output (18 lines):
  Traceback (most recent call last):
...
    File "setup.py", line 16, in <module>
      from build_helpers.build_helpers import (
  ModuleNotFoundError: No module named 'build_helpers'
...
```

This issue does not occur if I download the files directly from github.
The difference is that the `build_helpers` folder is included in the repository downloaded from github, but `build_helpers` is not present in the `.tar.gz` file I found on pypi.

**Comment by Jasha10** [comment]
This is closed in the new OmegaConf 2.1.1 release
```text
$ wget https://files.pythonhosted.org/packages/91/80/c9268fedb0d63ddf7dfa4352070f8cfecf2b0443df7737462c3ab0437c11/omegaconf-2.1.1.tar.gz
...
$ tar -xzf omegaconf-2.1.1.tar.gz
$ cd omegaconf-2.1.1/
$ pip install .
...
Successfully installed PyYAML-5.4.1 antlr4-python3-runtime-4.8 omegaconf-2.1.1
```
